### PR TITLE
#10 Provided Adapters API

### PR DESCRIPTION
**Pull request by serjiokov**

### CODE REVIEW

File: bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/BrickAdapter.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *	Sergei Kovalchuk <sergei.kovalchuk@arsysop.ru> - 
+ *												initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chronograph.api.adapters;
+
+import java.util.List;
+
+import org.eclipse.chronograph.api.Group;
+
+public interface BrickAdapter<T, S> {
```
Comment (ruspl-afed) [inline bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/BrickAdapter.java:20]: javadoc: what it for, info about template types

File: bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/GroupAdapter.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ *	Sergei Kovalchuk <sergei.kovalchuk@arsysop.ru> - 
+ *												initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chronograph.api.adapters;
+
+import java.util.List;
+
+import org.eclipse.chronograph.api.Section;
+
+public interface GroupAdapter<T, S> {
```
Comment (ruspl-afed) [inline bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/GroupAdapter.java:20]: javadoc: what it for, info about template types

File: bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/SectionAdapter.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ *
+ *	Contributors:
+ *	Sergei Kovalchuk <sergei.kovalchuk@arsysop.ru> - 
+ *												initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chronograph.api.adapters;
+
+import java.util.List;
+
+public interface SectionAdapter<T, S> {
```
Comment (ruspl-afed) [inline bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/SectionAdapter.java:18]: javadoc: what it for, info about template types

File: bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/SubGroupAdapter.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+ *	Sergei Kovalchuk <sergei.kovalchuk@arsysop.ru> - 
+ *												initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chronograph.api.adapters;
+
+import java.util.List;
+
+import org.eclipse.chronograph.api.Group;
+
+public interface SubGroupAdapter<T, S> {
```
Comment (ruspl-afed) [inline bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/SubGroupAdapter.java:20]: javadoc: what it for, info about template types

File: bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/BrickAdapter.java (hunk 1 above)
Comment (serjiokov) [reply bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/BrickAdapter.java]: updated

File: bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/SectionAdapter.java (hunk 3 above)
Comment (serjiokov) [reply bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/SectionAdapter.java]: updated

File: bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/SubGroupAdapter.java (hunk 4 above)
Comment (serjiokov) [reply bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/SubGroupAdapter.java]: updated

**Pull request merged**